{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -5,29 +5,109 @@
 #################################################################################
 
 BUCKET = {{ cookiecutter.s3_bucket }}
+PROJECT_NAME = {{ cookiecutter.repo_name }}
+PYTHON_INTERPRETER = {{ cookiecutter.python_interpreter }}
 
 #################################################################################
 # COMMANDS                                                                      #
 #################################################################################
 
-requirements:
+## Install Python Dependencies
+requirements: test_environment
 	pip install -q -r requirements.txt
 
+## Make Dataset
 data: requirements
 	python src/data/make_dataset.py
 
+## Delete all compiled Python files
 clean:
 	find . -name "*.pyc" -exec rm {} \;
 
+## Lint using flake8
 lint:
 	flake8 --exclude=lib/,bin/,docs/conf.py .
 
+## Upload Data to S3
 sync_data_to_s3:
 	aws s3 sync data/ s3://$(BUCKET)/data/
 
+## Download Data from S3
 sync_data_from_s3:
 	aws s3 sync s3://$(BUCKET)/data/ data/
 
+## Set up python interpreter environment
+create_environment:
+	source /usr/local/bin/virtualenvwrapper.sh; mkvirtualenv $(PROJECT_NAME) --python=$(PYTHON_INTERPRETER)
```

Comment:
Yep, agreed. Let's output the required profile lines to the console so users see them. I don't think there's a really elegant way to do this otherwise.

In general, for users who are opinionated about their environment tool, it should be easy to change. For those who care less, using the default tool in CCDS should be as painless as possible. Right now, I think that virtualenvwrapper is the least painful (primarily in that it does not require the Anaconda distribution of Python).